velox/functions/prestosql/window/tests/NthValueTest.cpp

```diff
@@ -152,5 +165,10 @@ TEST_F(NthValueTest, offsetValues) {
       {vectors}, "nth_value(c0, c2)", overClause, offsetError);
 }
 
+VELOX_INSTANTIATE_TEST_SUITE_P(
```

Comment:
Let's not iterate this over frames. I feel the frames test framework should be part of WindowTestBase functions.

Reply:
Sure, thanks for the suggestion!